app/views/hyrax/dashboard/collections/_document_list.html.erb

```diff
@@ -6,6 +6,7 @@
   <% @document_list = documents %>
   <% @disable_select_all = (@document_list.count <= 1) %>
   <% if params[:action] == "edit" %>
+    <!-- TODO: verify then remove? this branch and lower partials shouldn't be getting executed any longer -->
```

Comment:
Is there a ticket for this?

Reply:
No, I intend to work on the cleanup as a separate commit, once I am sure the rest of the PR meets @elrayle 's intent. That's why I requested her review of it.